src/abe_policy/policy.rs

```diff
@@ -167,8 +166,7 @@ impl Policy {
     /// Fails if there is no such attribute.
     fn get_attribute(&self, attr: &Attribute) -> Result<&AttributeParameters, Error> {
         if let Some(dim) = self.dimensions.get(&attr.dimension) {
-            dim.attributes
-                .get(&attr.name)
+            dim.get_attribute(&attr.name)
```

Comment:
Maybe rename this to `get` to follow the `HashMap` conventions:
```suggestion
            dim.get(&attr.name)
```